File: .js/8kyu-Grasshopper-Personalized-Message.js
// Create a function that gives a personalized greeting. This function takes two parameters: name and owner.

// Use conditionals to return the proper message:

// case	return
// name equals owner	'Hello boss'
// otherwise	'Hello guest'

describe("Tests", () => {
    it("test", () => {
  Test.assertEquals(greet('Daniel', 'Daniel'), 'Hello boss')
  Test.assertEquals(greet('Greg', 'Daniel'), 'Hello guest')
    });
  });

  
  // my code

  function greet (name, owner) {
    return (name == owner) ? `Hello boss` : `Hello guest`
  }

File: CodeWars/.cs/8kyu-Grasshopper-PersonalizedMessage.cs
// Create a function that gives a personalized greeting. This function takes two parameters: name and owner.

// Use conditionals to return the proper message:

// case	return
// name equals owner	'Hello boss'
// otherwise	'Hello guest'

namespace Solution 
{
  using NUnit.Framework;
  using System;

  [TestFixture]
  public class SolutionTest
  {
    [Test]
    public void MyTest()
    {
      Assert.AreEqual("Hello boss", Kata.Greet("Daniel", "Daniel"));
      Assert.AreEqual("Hello guest", Kata.Greet("Greg", "Daniel"));
    }
  }
}

// my code

public class Kata
{
  public static string Greet(string name, string owner) => (name == owner) ? "Hello boss" : "Hello guest";
}